### PR TITLE
image: use deletion handling metanamespace key in import scheduler

### PR DESCRIPTION
**Pull request by mfojtik**
@soltysh just bring the scheduled importer controller in sync with other controllers

### CODE REVIEW

File: pkg/image/controller/scheduled_image_controller.go [hunk 1]
```diff
@@ -92,22 +97,12 @@ func (s *ScheduledImageStreamController) updateImageStream(old, cur interface{})
...
-		if !isImageStream {
-			glog.V(2).Infof("tombstone contained unexpected object %#v", tombstone)
-			return
-		}
-	}
-	key, err := cache.MetaNamespaceKeyFunc(stream)
+	key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 	if err != nil {
-		glog.V(2).Infof("unable to get namespace key function for stream %s/%s: %v", stream.Namespace, stream.Name, err)
+		stream, err := meta.Accessor(obj)
```
Comment (deads2k) [inline pkg/image/controller/scheduled_image_controller.go]: just utilruntime.HandleError(err) and return

**Comment by deads2k** [comment]
nit.  lgtm otherwise.

**Comment by mfojtik** [comment]
nit fixed

**Comment by soltysh** [comment]
> @soltysh just bring the scheduled importer controller in sync with other controllers

Should that be used in all controllers, do we have an issue tracking this?

**Pull request merged**